st3/mdpopups/st_code_highlight.py

```diff
@@ -250,6 +257,12 @@ def set_view(self, src, lang, plugin_map):
                         break
             if loaded:
                 break
+        if not loaded:
+            # Use "source.LANG" and "text.LANG" as fallbacks if possible
```

Comment:
Can you add `fallbacks` to https://github.com/facelessuser/sublime-markdown-popups/blob/master/docs/src/dictionary/en-custom.txt?

Reply:
ok